src/addons/InfoBox.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import SimpleChildComponent from "../internals/SimpleChildComponent";
+import createRegisterEvents from "../internals/createRegisterEvents";
+import exposeGetters from "../internals/exposeGetters";
+import InfoWindow from "../InfoWindow";
+
+const {PropTypes} = React;
+
+class InfoBox extends SimpleChildComponent {
+  constructor (...args) {
+    super(...args);
+    this.state = {};
+  }
+
+  _createOrUpdateInstance () {
```

Comment:
Anyway, I'll do some refactor over `SimpleChildComponent#_createOrUpdateInstance` API and fix this issue. Please leave it as is, thanks!